docs/source-control/new-repo.md

```diff
@@ -35,7 +35,8 @@ and plugins ) don't need this and the default branch will be `main`.
 ## Branch Rules & Setup
 
 * For default branch
-    * Require pull request reviews before merging: Require at least 2 reviews
+    * Require a pull request before merging
```

Comment:
Thank you @rlmead for doing this! Would it be better to clarify that we need 2 dev and 1 qa approval before merging? Context is [here](https://theartofed.slack.com/archives/C0190RSMPQX/p1625080118213200?thread_ts=1625079465.210400&cid=C0190RSMPQX)

Reply:
Ah! Per a convo with @rlmead, we found relevant information regarding this [here](https://docs.theartofeducation.io/pages/source-control/code-reviews#code-review-approval), so additional clarification is not necessary.